core/store/src/test/java/uk/gov/gchq/gaffer/store/StoreTest.java

```diff
@@ -307,15 +296,16 @@ public void shouldHandleMultiStepOperations() throws Exception {
         final CloseableIterable<Element> getElementsResult = mock(CloseableIterable.class);
 
         final AddElements addElements1 = new AddElements();
-        final GetElementsBySeed<ElementSeed, Element> getElementsBySeed = new GetElementsBySeed<>();
+        final GetElements<ElementSeed, Element> getElementsBySeed = new GetElements<>();
         final OperationChain<CloseableIterable<Element>> opChain = new OperationChain.Builder()
                 .first(addElements1)
                 .then(getElementsBySeed)
                 .build();
 
 
         given(addElementsHandler.doOperation(addElements1, context, store)).willReturn(null);
-        given(getElementsHandler.doOperation(getElementsBySeed, context, store)).willReturn(getElementsResult);
+        given(getElementsHandler.doOperation(getElementsBySeed, context, store))
```

Comment:
Perhaps do a quick global search for getElementsBySeed and replace it with getElements.